drivers/video/video_stm32_dcmi.c

```diff
@@ -375,10 +375,12 @@ static int video_stm32_dcmi_set_frmival(const struct device *dev, struct video_f
 		.format = &data->fmt,
 	};
 	struct video_frmival best_sensor_frmival;
-	uint64_t best_diff_nsec = INT32_MAX;
-	uint64_t diff_nsec = 0, a, b;
+	uint32_t best_diff_nsec = INT32_MAX;
+	uint32_t diff_nsec = 0, a, b;
```

Comment:
Since values are computed in µs, I'd suggest the rename the variables (`best_diff_us`, `diff_us`) to prevent confusion.